### PR TITLE
updated install file to include mysql 8 support

### PR DESCRIPTION
**Pull request by alexanderjulianmartinez**
Added logic to use percona_xtrabackup_80 for MySQL 8 hosts to install.pp file

### CODE REVIEW

File: manifests/install.pp [hunk 1]
```diff
@@ -3,7 +3,11 @@
 # Class to install xtrabackup from packages
 #
 class xtrabackup::install {
-  if $::lsbdistcodename != 'lucid' {
+  if $xtrabackup::compatibility__version == 'mysql80' {
```
Comment (mtbentley) [inline manifests/install.pp]: extra underscore here?

Comment (alexanderjulianmartinez) [reply manifests/install.pp]: yes good catch, removed the extra character. Thank you!

File: manifests/install.pp [hunk 2]
```diff
@@ -3,7 +3,11 @@
 # Class to install xtrabackup from packages
 #
 class xtrabackup::install {
-  if $::lsbdistcodename != 'lucid' {
+  if $xtrabackup::compatibility_version == 'mysql80' {
+    package {'percona-xtrabackup-80':
+      ensure => $xtrabackup::version
+    }
+  } elsif $::lsbdistcodename != 'lucid' {
```
Comment (mtbentley) [inline manifests/install.pp]: we can also remove the lucid check here and just make this `else`

Comment (alexanderjulianmartinez) [reply manifests/install.pp]: I'm not sure that this corner cases is no longer in play, @VinaySagarGonabavi do you have further insight into this?

Comment (VinaySagarGonabavi) [reply manifests/install.pp]: There are no hosts with the ubuntu version lucid anymore. I don't think this is relevant anymore:
```
$ inv find -F lsbdistcodename=lucid

gonabavi@dev60-uswest1adevc:~/pg 2020-11-10 14:41:09

$
```

Comment (alexanderjulianmartinez) [reply manifests/install.pp]: fixed per conversation in slack

**Review by VinaySagarGonabavi — APPROVED** [review]

**Review by flopex — APPROVED** [review]

**Review by mtbentley — APPROVED** [review]

**Pull request merged**